chroma-manager/chroma_core/migrations/0039_add_primary_umtm_stepresults.py

```diff
@@ -6,10 +6,13 @@ class Migration(DataMigration):
     def forwards(self, orm):
         # update selected stepresults to include 'primary' arg (arbitrary value)
         for obj in orm['chroma_core.stepresult'].objects.all():
-            if obj.step_klass.__name__ == 'UpdateManagedTargetMount' \
-                    and 'primary' not in obj.args:
-                obj.args['primary'] = True
-                obj.save()
+            try:
+                if obj.step_klass.__name__ == 'UpdateManagedTargetMount' \
+                        and 'primary' not in obj.args:
+                    obj.args['primary'] = True
+                    obj.save()
+            except AttributeError:
```

Comment:
Are we skipping data in a migration by doing this?

Reply:
as far as I understand the UpdateManagedTargetMount steps that we are interested for this data migration will be exposed as objects that have __name__ attribute and therefore will not be skipped